### DISCUSSION TITLE
Laravel Dusk does not render the success message when filling in the form

### DISCUSSION THREAD

**Discussion by GijsGoudzwaard** [spatie/laravel-support-bubble#28] [Bugs]
The success message does not render when testing the support bubble with laravel dusk. When filling in the form as a user then it does render normally. So it only happens inside the tests. Using the code below with the default dusk configuration.

```php
public function test_we_can_submit_the_form(): void
{
    $this->browse(function (Browser $browser) {
        $browser->visit('/')
            ->click('.spatie-support-bubble > .spatie-support-bubble__button > button')
            ->waitFor('.spatie-support-bubble')
            ->type('#support-bubble-name', $this->faker->name)
            ->type('#support-bubble-email', $this->faker->safeEmail)
            ->type('#support-bubble-message', $this->faker->realText)
            ->press(__('support-bubble::support-bubble.submit_label'))
            ->waitForText(__('support-bubble::support-bubble.success'));
    });
}
```

Here is a screencast where you can see it happen. When it shows the white box it should be showing the success message, but it does not render anything. There are no errors in the console and the HTTP request returns a HTTP status of 200.

https://user-images.githubusercontent.com/5947260/165600855-b039a09c-8450-45ae-8b7c-96005331927c.mov

**Comment by freekmurze** [comment]
I'd accept a PR that fixes this 👍

**Comment by AlexVanderbist** [comment]
It looks like this might be because of the spam prevention: the `spatie/laravel-honeypot` package used in the support bubble popup will discard any submissions if they're sent too fast (as if by automated tools or bots 😅)
